x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/layout_factory.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { KbnServer } from '../../../../../../../../src/server/index';
+import { LayoutTypes } from '../../../common/constants';
+import { Layout } from './layout';
+import { PreserveLayout } from './preserve_layout';
+import { PrintLayout } from './print_layout';
+
+// you'll notice that we aren't passing the zoom at this time, while it'd be possible to use
+// window.pixelDensity to figure out what the current user is seeing, if they're going to send the
+// PDF to someone else, I can see there being benefit to using a higher pixel density, so we're
+// going to leave this hard-coded for the time being
+
+interface LayoutParams {
+  id: string;
+  dimensions: {
+    height: number;
+    width: number;
+  };
+}
+
+export function createlayout(server: KbnServer, LayoutParamsin: LayoutParams): Layout {
```

Comment:
`LayoutParamsin` => `layoutParams`.  

Should start with a lowercase since it's a variable, not a class name or type.